pkg/controller/apiclient/apiclient_util_test.go

```diff
@@ -14,18 +14,180 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package apiclient
+package apiclient_test
```

Comment:
seems like maybe the sane thing to do would be to have proper go package for apiclient utils in apiclient/util.